docs/release-notes/8.6.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[release-notes-header-8.6.0]]
+== 8.6
+
+[discrete]
+[[release-notes-8.6.0]]
+=== 8.6.0
+
+[discrete]
+[[known-issue-8.6.0]]
+==== Known issues
+There are no known issues in 8.6.0.
+
+[discrete]
+[[breaking-changes-8.6.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+//:pull: {pull}
+// end::breaking-changes[]
+There are no breaking changes in 8.6.0.
+
+
+[discrete]
+[[deprecations-8.6.0]]
+==== Deprecations
+There are no deprecations in 8.6.0.
+
+
+[discrete]
+[[features-8.6.0]]
+==== Features
+* Enables you to create new cases based on indicators, or directly add indicators to existing cases ({pull}145121[#145121]).
+* Adds the "is one of" operator to the *Add field* menu in Timeline ({pull}144988[#144988]).
+* Adds an "Add to timeline investigation" button to the User Risk Scores and Host Risk Scores cards on the Entity Analytics dashboard ({pull}144819[#144819]).
+* When duplicating a rule, allows you to choose whether the duplicate should include the original rule exceptions ({pull}144782[#144782]).
+* Improves the Rule exceptions interface and allows you to export read-only exception lists ({pull}144383[#144383]).
+* Enables a technical preview of functionality that enables users to create run time queries that can be parameterized from alert data, or hard coded literal values. ({pull}145240[#145240]).
+* Enables you to schedule Osquery `global packs` ({pull}143948[#143948]).
+* Creates a new connector for Tines ({pull}143505[#143505]).
+* Updates the UI for adding and editing exceptions ({pull}143127[#143127]).
+* Creates an "Exception list details" page, where you can view linked rules and modify the exception list ({pull}143041[#143041]).
+* Enables you to bulk-add events to Timeline (up to 4000) ({pull}142737[#142737]).
+* Enables alert throttling per rule execution for query and saved query rules ({pull}142686[#142686]).
+* Restricts access to the Response Actions History page based on Kibana `Actions Log Management` permissions ({pull}142825[#142825]).
+
+[discrete]
+[[bug-fixes-8.6.0]]
+==== Bug fixes and enhancements
+* Adds a link to *View indicators* to the Threat Intelligence card ({pull}145125[#145125]).
+* Improves the interface for creating rule exceptions and shared exception lists ({pull}144575[#144575]).
+* Adds cases metadata to the alert alert details page, in the Cases panel ({pull}144430[#144430]).
+* Updates Osquery schema to v5.5.1 ({pull}144090[#144090]).
```

Comment:
```suggestion

```

Reply:
This likely belongs in the Kibana release notes. I can move this there when I get back or you can ping @KOTungseth for assistance getting it in. :)